.ci/scripts/distribution/random-test-java.sh

```diff
@@ -14,7 +14,6 @@ MAVEN_PROPERTIES=(
   -Dsurefire.rerunFailingTestsCount=0
   -Dmaven.javadoc.skip=true
 )
-tempFile=$(mktemp)
```

Comment:
:question: Do you remember why we used a temporary file for the test output?

Reply:
It was a copy paste left over. You can see in the other `test-java.sh` for example, it's used to later analyze flaky tests. But since the flaky test plugin doesn't work with randomized tests here it's not used, so tee'ing to the temp file was unnecessary.